base_global_discount/models/global_discount.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class GlobalDiscount(models.Model):
+    _name = 'global.discount'
+    _description = 'Global Discount'
+    _order = "sequence, id desc"
+
+    sequence = fields.Integer(
+        help='Gives the order to apply discounts',
+    )
+    name = fields.Char(
+        string='Discount Name',
+        required=True,
+    )
+    color = fields.Integer('Color Index', default=10)
```

Comment:
Why this?

Reply:
In a first moment I thought it could be nice and cheap to have it in the m2m tags but in a second though it looks lame in this case. Removing it